### ISSUE TITLE
Request: Save and Load merkle tree from file

### ISSUE THREAD

**Issue by russick89** [miguelmota/merkletreejs#44]
I am generating a merkle tree with a few hundred thousand leaves. Once I have the final tree, I would like to save it to a file and reload it.  So far I've generated the leaves and saved them to a JSON file and that worked fine, but it still takes a while and a decent amount of ram to generate the tree.  Ideally, I would like to generate the final tree (not just the leaves) and save it to a JSON file so that I can quickly load it next time I need it instead of generating the entire thing.

Any ideas on how to achieve this?

**Comment by andrew-sol** [comment]
I'm also wondering how to serialize/deserialize the tree.

**Comment by ductridev** [comment]
This should be marked as feature!

**Comment by Montoya** [comment]
I also need this feature

**Comment by xtools-at** [comment]
yeah, or just parsing it from a hex string would be great already, something like `new MerkleTree("0x256")`.

**Comment by Shanni** [comment]
requesting this feature +1
